app/api/helpers/export_helpers.py

```diff
@@ -175,7 +176,7 @@ def export_event_json(event_id, settings):
     Exports the event as a zip on the server and return its path
     """
     # make directory
-    dir_path = 'static/exports/event%d' % event_id
+    dir_path = app.config['BASE_DIR'] + '/static/uploads/event%d' % event_id
```

Comment:
Create a new dir inside uploads called as `exports` and put it there. 

(Be sure to create that directory if it doesn't already exist)